src/lib/connectTraceToPlot.js

```diff
@@ -42,6 +46,7 @@ export default function connectTraceToPlot(WrappedComponent) {
         container: trace,
         fullContainer: fullTrace,
       };
+      this.type = plotlyTraceToCustomTrace(trace);
```

Comment:
@bpostlethwaite // @VeraZab let me know if you think there is a better place for this to live

Reply:
I think its fine here